presentation/src/main/java/me/androidbox/presentation/agenda/viewmodel/AgendaViewModel.kt

```diff
@@ -243,4 +248,14 @@ class AgendaViewModel @Inject constructor(
     private fun syncFullAgendaItems() {
         fullAgendaSynchronizer.sync()
     }
+
+    private fun synAgendaTasksReminders() {
+        taskReminderSynchronizer.sync()
+    }
```

Comment:
You are naming it "taskReminderSynchronizer" when in reality you sync everything. Not just tasks or reminders

Reply:
@MKiperszmid Will refactor the naming